modules/readme/src/main/tut/README.md

```diff
@@ -1,9 +1,3 @@
-[comment]: # (Start Badges)
```

Comment:
@juanpedromoreno can you weigh in on this and verify if it's the correct to remove these markers?

Reply:
Yeah, as far as I remember, the plugin will perform the badge updates the root's README. So I guess we might not need those markers there.

Reply:
I think you mentioned before that we needed an empty newline at the beginning of the file? Is that still the case?